recipes/polly/build.sh

```diff
@@ -0,0 +1,5 @@
+cd polly
+mkdir -p build && cd build
+cmake -DCMAKE_BUILD_TYPE="Release" -DLLVM_ENABLE_PIC=1 -DCMAKE_INSTALL_PREFIX=${PREFIX} ..
+make -j${CPU_COUNT}
```

Comment:
Suggest to break the CMake line and be explicit about the generator. Also a bit less crammed vertically.

```suggestion
mkdir -p build && cd build

cmake -G "Unix Makefiles" \
    -DCMAKE_BUILD_TYPE="Release" \
    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
    -DLLVM_ENABLE_PIC=1 \
    ..

make -j${CPU_COUNT}
```